src/components/TimeCardPage/CellTypes/CommentCell.tsx

```diff
@@ -1,43 +1,196 @@
-import moment from 'moment-timezone';
-import React, {useState, useEffect} from 'react'; 
+import React, {useState, useEffect, useContext} from 'react'; 
+import {Button} from '@chakra-ui/react';
+import { UserContext } from '../UserContext';
+import {
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    ModalCloseButton,
+    Input,
+    IconButton,
+    ButtonGroup,
+    Flex,
+    useDisclosure,
+    useEditableControls
+  } from '@chakra-ui/react';
+import {
+    ChatIcon,
+    WarningIcon,
+    CheckIcon,
+    CloseIcon,
+    EditIcon
+  } from '@chakra-ui/icons';
+import {
+  Editable,
+  EditableTextarea,
+  EditablePreview,
+  EditableInput
+  } from '@chakra-ui/react';
 
 import {CommentSchema} from '../../../schemas/RowSchema'; 
-import {CommentType} from '../types'; 
-import {TIMEZONE} from '../../../constants'; 
+import {CommentType, CellStatus} from '../types'; 
+import CommentModal from '../CommentModal'
 
 interface CommentProps {
     comments: CommentSchema[] | undefined; 
     setComment: Function; 
 } 
 
+
+// setComments can be used for comments or reports
+interface ShowCommentModalProps{
+    comments: CommentSchema[];
+    setComments: Function;
```

Comment:
❓What does the `Function` type represent? I usually just do something like `setComments: (comments: CommentSchema[]) => void`